modules/disabling-node-caching-in-elyra.adoc

```diff
@@ -4,7 +4,7 @@
 = Disabling node caching in Elyra
 
 [role='_abstract']
-Elyra offers a feature within {productname-short} that allows you to cache components, or "nodes," within your data pipelines. When a pipeline component runs, Elyra stores its outputs by default. In subsequent runs, if Elyra detects that a particular component has already been executed and its inputs have not changed, it reuses the cached outputs instead of re-running the entire component. 
+Elyra offers a feature within {productname-short} that allows you to cache components, or "nodes", within your data pipelines. When a pipeline component runs, Elyra stores its outputs by default. In subsequent runs, if Elyra detects that a particular component has already been executed and its inputs have not changed, it reuses the cached outputs instead of re-running the entire component. 
```

Comment:
```suggestion
You can use the Elyra feature in {productname-short} to cache components, or "nodes", within your data pipelines. When a pipeline component runs, Elyra stores its outputs by default. In subsequent runs, if Elyra detects that a particular component has already been executed and its inputs have not changed, it reuses the cached outputs instead of re-running the entire component. 
```